libs/environment/src/external-chain.ts

```diff
@@ -53,3 +53,11 @@ export function getExternalChainLabel(chainId?: string) {
     return 'Custom Chain';
   }
 }
+
+export function getExternalChainShortLabel(chainId?: string) {
+  if (chainId && SUPPORTED_CHAIN_IDS.includes(chainId)) {
+    return SUPPORTED_CHAIN_SHORT_LABELS[chainId];
+  } else {
+    return '';
```

Comment:
**nitpick (non-blocking)**:  as per the previous comment, I'd just return `chainId` rather than blank string, as it's better than nothing.

Reply:
done